pyanaconda/kickstart.py

```diff
@@ -1516,12 +1516,12 @@ def execute(self, storage, ksdata, instClass):
             raidmems.append(dev)
 
         # Now get a format to hold a lot of these extra values.
-        kwargs["fmt"] = getFormat(ty,
+        kwargs["fmt"] = get_format(ty,
            label=self.label,
            fsprofile=self.fsprofile,
            mountpoint=self.mountpoint,
            mountopts=self.fsopts,
-           createOptions=self.mkfsopts)
+           create_options=self.mkfsopts)
         if not kwargs["fmt"].type:
             raise KickstartParseError(formatErrorMsg(self.lineno,
                     msg=_("The \"%s\" file system type is not supported.") % ty))
```

Comment:
`kwargs["memberDevices"]` and `kwargs["totalDevices"]` on lines 1532 and 1533 should be `kwargs["member_devices"]` and `kwargs["total_devices"]` (GitHub won't allow me to add a comment to unchanged lines...)